nixos/modules/services/backup/btrbk.nix

```
@@ -232,12 +271,15 @@ in
       cfg.instances;
     systemd.services = mapAttrs'
       (
-        name: _: {
+        name: instance: {
           name = "btrbk-${name}";
           value = {
             description = "Takes BTRFS snapshots and maintains retention policies.";
             unitConfig.Documentation = "man:btrbk(1)";
-            path = [ "/run/wrappers" ] ++ cfg.extraPackages;
+            path = [ "/run/wrappers" ]
+              ++ cfg.extraPackages
+              ++ (if instance.settings.stream_compress == "no" then []
+                else [ (getAttr instance.settings.stream_compress streamCompressMap) ]);
```

```suggestion
              ++ lib.optional (instance.settings.stream_compress == "no") (getAttr instance.settings.stream_compress streamCompressMap);
```
we can shorten that a bit